serenata_toolbox/federal_senate/federal_senate_dataset.py

```diff
@@ -1,56 +1,93 @@
 import os.path
+from datetime import date
 from urllib.request import urlretrieve
-import numpy as np
+
 import pandas as pd
 
-from datetime import date
 
 class FederalSenateDataset:
     URL = 'http://www.senado.gov.br/transparencia/LAI/verba/{}.csv'
-    FIRST_YEAR = 2008
-    NEXT_YEAR = date.today().year + 1
 
-    YEAR_RANGE = range(FIRST_YEAR, NEXT_YEAR)
+    LAST_YEAR = date.today().year + 1
 
-    def __init__(self, path):
+    def __init__(self, path, first_year=2008, last_year=LAST_YEAR):
         self.path = path
+        self.first_year = first_year
+        self.last_year = last_year
+        self.year_range = range(first_year, last_year)
 
     def fetch(self):
-        for year in self.YEAR_RANGE:
+        retrieved_files = []
+        not_found_files = []
+
+        for year in self.year_range:
             url = self.URL.format(year)
             file_path = os.path.join(self.path, 'federal-senate-{}.csv'.format(year))
-            urlretrieve(url, file_path)
+            try:
+                urlretrieve(url, file_path)
+                retrieved_files.append(file_path)
+            except Exception as exception:
+                print('While fetching Seranata Toolbox not found file: {0} \n{1}'.format(file_path, exception))
```

Comment:
No need to number the `{…}` in the Python version we use (actually in anythig higher than Python 3.0. Can you cleanup?